storage/innobase/btr/btr0btr.cc

```diff
@@ -1192,17 +1192,26 @@ btr_create(
 /** Free a B-tree except the root page. The root page MUST be freed after
 this by calling btr_free_root.
 @param[in,out]	block		root page
-@param[in]	log_mode	mtr logging mode */
+@param[in]	log_mode	mtr logging mode
+@param[in]	is_ahi_allowed	false for intrinsic tables else true */
```

Comment:
argument name and description differ. Perhaps the description explains when AHI is allowed, but that should be made clear

Reply:
ahi is not applicable/allowed for intrinsic tables, For non-intrinsic tables, AHI is applicable/allowed.

Didn't get the question. What should I change here?